Extension/src/LanguageServer/configurations.ts

```diff
@@ -1377,6 +1377,13 @@ export class CppProperties {
             result = result.replace(/\*/g, "");
         }
 
+        // Make sure all paths result to an absolute path
+        if (!path.isAbsolute(result)) {
```

Comment:
Can you combine the "if"? i.e. `if (!path.isAbsolute(result) && this.rootUri) {`?